src/locales/en/translation/manage-users.json

```diff
@@ -5,6 +5,7 @@
     "back_link": "Return to authorised agent services",
     "email_address": "Email address",
     "page_header": "Manage users",
+    "page_header_standard": "View users",
```

Comment:
I think we'd need to make the same entry in the Welsh equivalent file:
https://github.com/companieshouse/acsp-manage-users-web/blob/4857ea960b482b8734f19444a11f1a924796b63a/src/locales/cy/translation/manage-users.json#L7-L8

Reply:
Even if it's just a placeholder:

```diff
diff --git a/src/locales/cy/translation/manage-users.json b/src/locales/cy/translation/manage-users.json
--- a/src/locales/cy/translation/manage-users.json	(revision 1189fba02e0a0b9abeefc8fc931c43c71390827d)
+++ b/src/locales/cy/translation/manage-users.json	(date 1722422627702)
@@ -5,6 +5,7 @@
     "back_link": "[CY] Return to authorised agent services",
     "email_address": "[CY] Email address",
     "page_header": "[CY] Manage users",
+    "page_header_standard": "[CY] View users",
     "remove": "[CY] Remove",
     "remove_user": "[CY] Remove user",
     "standard_users": "[CY] Standard users",
```